### ISSUE TITLE
2021-12-9

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#679]
121IV.3. Compared with Distillation
122Read code of own-teacher; 
123Read train.py;  parser.add_argument('cmd', choices=['train', 'test']) ; done; '--data-dir', default='/home/xxx/cifar100' ; --dataset', '-d', type=str, default='cifar100',                         choices=['cifar10', 'cifar100'] ; done; '--epoch', default=200, type=int,                         help='number of total iterations (default: 64,000)' ; done; '--batch-size', default=128, type=int,                         help='mini-batch size (default: 128)' ; done; '--lr', default=0.1, type=float,                         help='initial learning rate' ; done; '--weight-decay', default=5e-4, type=float,                         help='weight decay (default: 1e-4)' ; done; '--print-freq', default=100, type=int,                         help='print frequency (default: 10)' ; done; '--temperature', default=3, type=int,                         help='temperature to smooth the logits' ; done; (2021-11-24)
124What is nn.Conv2d ? Applies a 2D convolution over an input signal composed of several input planes ; Pytorch 从0开始学（6）——Conv2d 详解 https://zhuanlan.zhihu.com/p/99742120 (2021-11-24)
125python单斜杠和双斜杠运算符的区别 ; 1、单斜杠（/）表示除法，且不管除数和被除数是不是整数，最后结果都是float类型。2、双斜杠（//）表示地板除，即先做除法（/），然后向下取整（floor）。至少有一方是float型时，结果为float型；两个数都是int型时，结果为int型。(2021-11-24)
126Python中，可包含其他对象的对象，称之为“容器”。容器是一种数据结构。常用的容器主要划分为两种：序列（如：列表、元祖等）和映射（如：字典）。序列中，每个元素都有下标，它们是有序的。映射中，每个元素都有名称（又称“ 键 ”），它们是无序的。除了序列和映射之外，还有一种需要注意的容器——“ 集合 ”。(2021-11-24)
127专家观点|人工智能解决数学题：“大力未必出奇迹” https://hub.baai.ac.cn/view/12125 (2021-11-25)
128What is nn.Sequential ? A sequential container. Modules will be added to it in the order they are passed in the constructor ; (2021-11-25)
129What is nn.BatchNorm2d ? Applies Batch Normalization over a 4D input (a mini-batch of 2D inputs with additional channel dimension) as described in the paper Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift ; (2021-11-25)
